games-arcade/ascii-invaders/ascii-invaders-1.0.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Space invaders clone, using ncurses library"
+HOMEPAGE="https://github.com/macdice/ascii-invaders"
```

Comment:
When you see something moved to github/gitlab and such, nice to have a separate commit to update metadata.xml like commit 445c58728bba63b0bb937416a1cfec699ffe5f82

Feel free to adjust indenting at same time to be more like the above commit (other devs may have their own preferences but that's at least how games@ normally does it).

And you can remove the longdescription, this doesn't really add anything over the short description. That this is compiling for Linux sure isn't news :)

Reply:
Ahh, yeah, you're right. Checking the `metadata.xml` file is something i easy forget sometimes... :D
I've updated it and removed the longdescription.